pages/docs/visualizing-data/period-comparisons.mdx

```diff
@@ -0,0 +1,18 @@
+import { Callout } from "nextra-theme-docs";
+
+
+# Period-over-period Comparisons
+
+Period-over-period comparisons allow you to compare timeseries data from different periods of time. They can be configured via the control panel on the right-hand side of the Data Explorer, under the `Compare` heading.
+
+Currently, Hashboard supports `Previous Year`, `Previous Month`, and `Previous Week` comparisons.
```

Comment:
Think this could afford an example or two demonstrating the business logic, specifically to show that this is all calendar based. Like: "For instance, when you apply a Previous Year comparison to a chart that is showing data by month for April 2022, May 2022, and June 2022, the comparison series will show data for April 2021, May 2021, and June 2021, respectively."